crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -305,7 +307,14 @@ def disable_non_core_proposals
     end
 
     # and finally delete the roles itself
-    active_non_core_roles.each(&:destroy)
+    roles_to_disable.each do |role|
+      if role.name == "crowbar-config-default"
+        role.elements["crowbar-upgrade"] = []
+        role.save
+      else
+        role.destroy
```

Comment:
This will also delete the "crowbar-upgrade" role. We shouldn't do that. I think ```active_non_core_roles.each(&:destroy)``` and updating crowbar-config-default is enough.